src/controllers/controller.bar.js

```diff
@@ -149,6 +149,11 @@ module.exports = DatasetController.extend({
 		var dataset = me.getDataset();
 		var options = me._resolveElementOptions(rectangle, index);
 
+		// float-bar support, if y arguments are array lets override rectangles styles, assigning no skippingBorder
+		if (helpers.isArray(dataset.data[index])) {
+			options.borderSkipped = null;
```

Comment:
We probably don't want to modify the `options`. I think you can do `rectangle._view.borderSkipped = null` to get the same result